src/Components/Play/Play.jsx

```diff
@@ -63,10 +63,18 @@ class Play extends Component {
 
         server.get(`/getgame/${gameID}`)
             .then(function(response) {
+                self.setState({loaded: true});
                 var players = response.data.players
-                if (Object.values(players).includes(user)) {
-                    var side = objectFlip(players)[user];
-                    self.setState({orientation: (side === 'w') ? 'white' : 'black'});
+
+                if (Object.values(players).includes(user) || (response.data.creator === user)) {
+                    if ((!response.data.public) && (parseInt(response.data.free_slots) > 0)) {
+                        alert(`There are open slots in this game! Send your friends the link:\n\nhttps://deeper-blue.me/invite/${self.props.location.pathname.split('/')[2]}\n\n`);
```

Comment:
Works but is annoying, I'd prefer an alert div at the top of the page. Not worth changing

Reply:
Maybe it is worth changing actually, just noticed that you can't even copy the link on the alert in Chrome.

Reply:
But yeah, other things to do